tests/test_email.py

```diff
@@ -135,6 +135,33 @@ def test_send_email_task_sendgrid_error(mock_sendgrid_client, mock_urlopen, db_s
     db_session.delete(history)
     db_session.commit()
 
+@pytest.mark.parametrize('input', [[{}], [{'name': 'Name'}], [{'email': 'Email'}]])
+@patch('urllib.request.urlopen')
+@patch('sendgrid.SendGridAPIClient')
+def test_send_email_task_invalid_email_obj(mock_sendgrid_client, mock_urlopen, input, db_session):
```

Comment:
❓: Was having issues with the order of the params - I wanted `input` to be first in the function params, but it seems like (even with changing the order of the decorators) that the patches seem to always inject afterwards... so this was the order I had to do to make ti work. Any suggestions?

Reply:
tagging @josh-chou to help with one and in general assist with reviewing the PR. =D

Reply:
Addressed in [commit](https://github.com/SFDigitalServices/email-microservice-py/pull/21/commits/42194d533ca357a47ae73b29965cbbcb7513a365).

Ended up splitting this out to a unit test as it lent itself to it nicely, and as a result, also sidestepped the question.